adoc/admin-centralized-logging.adoc

```diff
@@ -0,0 +1,212 @@
+for example, Rsyslog server.
+
+Collecting logs in a central location can serve for audit or debug purposes or to analyze and visually present data.
+
+== Types of logs
+
+You can log the following groups of services. See the <<Deployment>>
+for more information on how to select and customize the logs.
+
+Kubernetes System Components::
+* Kubelet
+* Cri-o
+
+Kubernetes Control Plane Components::
+* API Server
+* Controller Manager
+* Scheduler
+* Cilium
+* Kube-proxy
+* all resources in the kube-system namespaces
+
+Kubernetes Namespaces Pods::
+* All namespaces in cluster except kube-system
+
+OS Components::
+* Kernel
+* Audit
+* Zypper
+* Network (wicked)
+
+Centralized logging is also restricted to the following protocols: UDP, TCP, TCP + TLS, TCP + mTLS.
+
+
+== Log formats
+
+The two supported syslog message formats are *RFC 5424* and *RFC 3164*.
+
+[NOTE]
+====
+The Kubernetes cluster metadata is included in the RFC 5424 message.
+====
+
+Example RFC 3164
+----
+2019-05-30T09:11:21.968458+00:00 worker1 k8s.system/crio[12080] level=debug msg="Endpoint successfully created" containerID=caa46f14a68e766b877af01442e58731845bb45d8ce1f856553440a02c958b2f eventUUID=e2405f2a-82ba-11e9-9a06-fa163eebdfd6 subsys=cilium-cni
+----
+
+Example RFC 5424
+----
+<133>1 2019-05-30T08:28:38.784214+00:00 master0 k8s.pod/kube-system/kube-apiserver-master0/kube-apiserver - - [kube_meta namespace_id=1e030def-81db-11e9-a62b-fa163e1876c9 container_name=kube-apiserver creation_timestamp=2019-05-29T06:29:31Z host=master0 namespace_name=kube-system master_url=https://kubernetes.default.svc.cluster.local:443 pod_id=4aaf10f9-81db-11e9-a62b-fa163e1876c9 pod_name=kube-apiserver-master0] 2019-05-30T08:28:38.783780355+00:00 stderr F I0530 08:28:38.783710       1 log.go:172] http: TLS handshake error from 172.28.0.19:45888: tls: client offered only unsupported versions: [300]
+----
+
+
+== Installing Helm and Tiller
+
+In order to successfully use Centralized Logging, you first need to install `Helm` and `Tiller`.
+Helm is used to install the log agents and provide custom logging settings.
+
+- As of {productname} {productversion},
+Helm is not part of the {productname} package repository, so to use Centralized Logging,
+install the {productname} 3 package to install the Helm client. Run the following command from
+the location where you normally run `skuba` commands:
+
+
+[source,bash]
+----
+sudo zypper addrepo http://download.suse.de/ibs/SUSE/Updates/SUSE-CAASP/3.0/x86_64/update/SUSE:Updates:SUSE-CAASP:3.0:x86_64.repo
+sudo zypper refresh
+sudo zypper install -y helm
+
+helm init --client-only
+----
+
+////
+Note: what follows here are the instutions to be included after the Helm package has been integrated into v4:
+To install Helm, use the common zypper command:
+----
+sudo zypper install helm
+----
+
+Then initialize the Helm client by running:
+----
+helm init --client-only
+----
+////
+
```

Comment:
@lcavajani could you please review this?

Reply:
The requests have been accepted in OBS so we'll have the package tomorrow.

I have uncommented the official procedure ;)